### ISSUE TITLE
[NTR] Mouse strain C57BL/6NCrl

### ISSUE THREAD

**Issue by mshadbolt** [EBISPOT/efo#1023]
For new term requests, please provide the following information:

## Preferred term label

C57BL/6NCrl

## Synonyms
C57BL/6NCrlBr, C57BL/6NCrlBr, C57BL/6NCrlBR

## Textual definition

The C57BL/6NCrl is an inbred mouse strain derived from stock given to Charles River from the NIH in 1974. 

adapted from:
https://www.criver.com/products-services/find-model/c57bl6-mouse?region=3616

Other potentially useful cross refs:
http://www.informatics.jax.org/strain/MGI:2683688
https://phenome.jax.org/strains/839

## Suggested parent term

EFO:0005181 C57BL

http://www.ebi.ac.uk/efo/EFO_0005181

## Attribution

0000-0001-7505-5418

This is requested to enable proper curation of a strain used in this study: https://www.nature.com/articles/s41467-020-17800-6#Sec6 that I am currently wrangling

**Comment by paolaroncaglia** [comment]
Hi @mshadbolt ,
Thanks. I'll add 'C57BL/6NCrl' in time for the next EFO release (April 15th), and will comment here with the new EFO ID when done.

Notes for self:
- [ ] Add subset tag “added_for_HCA”.
- [ ] The first 2 synonyms are identical, so I’ll only include one (the error comes from MGI, I notified them).
- [ ] Of the 3 references provided for the strain, I’ll only include http://www.informatics.jax.org/strain/MGI:2683688 as it’s the most complete and reliable (lasting in time).
- [ ] The article provided is PMID:32788667.

**Comment by paolaroncaglia** [comment]
Hi again @mshadbolt ,
The new term EFO_0011030 (C57BL/6NCrl) will be in tomorrow's EFO release, and in the HCAO release scheduled for April 19th. Thanks,
Paola
